src/constants/index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    serverConfig: {
+        port: 4200,
+        hostname: 'http://localhost'
+    },
+    emittedEvents: {
+        userDisconnected: 'user_disconnected',
```

Comment:
You could use [symbols](https://developer.mozilla.org/en-US/docs/Glossary/Symbol) for events